src/ui-strings.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { addStrings } from '@wordpress/components';
+import { select } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+export const uiStrings = () => {
+	const { getSettings } = select( 'core/block-editor' );
+	const unsupportedBlockEditor = {
+		supported:
+			getSettings( 'capabilities' ).unsupportedBlockEditor === true,
+		switchable:
+			getSettings( 'capabilities' ).unsupportedBlockEditorSwitch === true,
+	};
```

Comment:
Nitpick: An alternative here could be to use:
```
	const { 
		unsupportedBlockEditor = false,
		unsupportedBlockEditorSwitch = false
	} = getSettings( 'capabilities' ).unsupportedBlockEditor;
```

Reply:
`unsupportedBlockEditor.supported` and `unsupportedBlockEditor.switchable` I think gives a nice context on their usage. Like:

```javascript
if (
 	unsupportedBlockEditor.supported === false &&
 	unsupportedBlockEditor.switchable
 )
```
vs
```javascript
if (
 	unsupportedBlockEditor === false &&
 	unsupportedBlockEditorSwitch
)
```

What do you think?

Reply:
(By the way, I think this whole file is a mess anyway, and not much extendible if we add more strings in the future. We will probably split it in the future)

Reply:
Ideally, those attributes (`unsupportedBlockEditor` and `unsupportedBlockEditorSwitch`) would be descriptive enough on their own so that they can be deconstructed and used as-is without needing to be renamed. But I know it's hard here to find a good name, so I'm happy to leave it as-is if you want to.